pkg/service/service.go

```diff
@@ -0,0 +1,216 @@
+package service
+
+import (
+	"context"
+	"time"
+
+	"github.com/boltdb/bolt"
+	"github.com/pkg/errors"
+	"github.com/segmentio/ksuid"
+	"github.com/vmihailenco/msgpack/v5"
+
+	"github.com/dlmiddlecote/ooohh"
+)
+
+var _ ooohh.Service = &service{}
```

Comment:
Why is this required? Somehow instantiating the interface or something?

Reply:
It isn’t required. It’s actually a test. I should move it there. 

It’s essentially a check that `service` does indeed satisfy the Service interface. 

This article has more on it: https://dave.cheney.net/2017/06/11/go-without-package-scoped-variables